python/lsst/obs/subaru/ccdTesting.py

```diff
@@ -1002,7 +986,7 @@ def grads(exp):
     plt.clf()
     for a in ccd:
         ima = im[a.getRawDataBBox()].getArray()
-        I = np.mean(ima, 1)/np.median(ima)
+        I = np.mean(ima, 1)/np.median(ima)  # noqa E741
```

Comment:
@PaulPrice not sure if you would rather rename this variable to fix the warning or keep the noqa.